tests/unit/Logger.js

```diff
@@ -176,7 +180,7 @@ describe('WereLogs Logger is usable:', () => {
     it('Uses the additional fields as expected', done => {
         const dummyLogger = new DummyLogger();
         const logger = new Logger('test');
```

Comment:
Hi David,
Thank you very much for the comments! At this point I think it'd be best if we have a call once your back, thanks again for your time and enjoy the rest of your vacation!